apps/cms/src/collections/weekly-newsletters/newsletter-button.tsx

```diff
@@ -101,9 +123,9 @@ const NewsletterButton = (): React.ReactElement => {
           ((e.target as HTMLElement).style.backgroundColor =
             buttonStyle.backgroundColor)
         }
-        onClick={() => void copyToClipboard()}
```

Comment:
This could probably just be changed from a button to a <a> tag with the download attribute, instead of doing it programmatically 🤔

But doesn't matter really, just a few less lines of code and better semantics

Reply:
Good idea! Now I changed it and the result is a lot cleaner. Imo this is now good for merge but some admin can check it